Update some API changes: (#3759)
timescale/docs: api/add_continuous_aggregate_policy.md

|Name|Type|Description|
|-|-|-|
|`if_not_exists`|BOOLEAN|Set to `true` to issue a notice instead of an error if the job already exists. Defaults to false.|

> |`timezone`|TEXT|A valid time zone. If `initial_start` is also specified, subsequent executions of the refresh policy will be aligned on its initial start. However, daylight savings time (DST) changes may shift this alignment. Set to a valid time zone if this is an issue you want to mitigate. If omitted, UTC bucketing is performed. Defaults to `NULL`.|

Reviewer: ```suggestion
|`timezone`|TEXT|A valid time zone. If you specify `initial_start`, subsequent executions of the refresh policy are aligned on `initial_start`. However, daylight savings time (DST) changes may shift this alignment. If this is an issue you want to mitigate, set `timezone` to a valid time zone. Default is `NULL`, [UTC bucketing](https://docs.timescale.com/use-timescale/latest/time-buckets/about-time-buckets/) is performed.|
```

> | `include_tiered_data` | BOOLEAN | Set true to read tiered data even if `timescaledb.enable_tiered_reads` is false. Defaults to `TRUE`. |

Reviewer 2: I thought the default is to use the current timescaledb.enable_tiered_data settings. @zilder ? 
```suggestion
| `include_tiered_data` | BOOLEAN | Enable/disable reading tiered data. This setting helps override the current settings for the`timescaledb.enable_tiered_reads` GUC. The default is NULL i.e we use the current setting for `timescaledb.enable_tiered_reads` GUC  | |
```

Reviewer 3: Yes, it defaults to NULL meaning the behavior is defined by the `timescaledb.enable_tiered_reads` configuration.
